### DISCUSSION TITLE
Tuesday 4th June: Outage to MOOSE 'PUT' 'GET' 'SELECT and 'FILTER' commands for system maintenance

### DISCUSSION THREAD

**Discussion by mo-roger-milton** [MetOffice/simulation-systems#203] [Monsoon]
Tuesday 4th June: Outage to MOOSE 'PUT' 'GET' 'SELECT and 'FILTER' commands for system maintenance to apply latest Oracle patch to production databases.

At 10:00 BST on Tuesday 4th June there will be an outage to MOOSE in order to shut down the Prod database in order to apply the latest Oracle patch. This is scheduled between 10:00 BST and 12:00 BST but may end sooner. All MOOSE 'PUT, 'GET', 'SELECT' and 'FILTER' commands will fail.
 
Change CHG0117312 refers.

**Comment by mo-mahammedk** [comment]
Following the system maintenance, we are aware some users are experiencing performance issues with MOOSE. It's very busy and is therefore causing slowdowns and errors for various jobs. We are looking into this and hope to get it resolved soon.
 
I will be in touch as soon as possible to let you know when normal service has been restored.
 
Apologies for any inconvenience!
